pages/start.tsx

```diff
@@ -107,243 +116,128 @@ const LoginStep1: React.FC<LoginStep1Props> = ({ handleLoginStep, user }) => {
-              </div>
-            </div>
-            <div className="flex gap-2 items-start">
-              <div className="mt-0.5 shrink-0">
-                <Icon IconImage={HighlightIcon} />
-              </div>
-              <div className="mt-0">
-                <Text className="!text-base !text-light-slate-12">
-                  Fetch basic GitHub information from Pull Requests and Issues
-                </Text>
-              </div>
-            </div>
-          </div>
-        </div>
-        <div className="flex flex-col gap-6">
-          <div>
-            <TextInput
-              placeholder="Organization Slug"
-              descriptionText="Grab the slug from GitHub, eg: 'open-sauced'."
-              classNames="!rounded-md !py-0.5 child:h-8 child:text-sm"
-              handleChange={handleSlugChange}
-            />
+          <div className="flex gap-3 flex-wrap">
+            {interestArray.map((topic, index) => (
+              <LanguagePill
+                onClick={() => handleSelectInterest(topic)}
+                classNames={`${(selectedInterest || []).includes(topic) && "bg-light-orange-10 text-white"}`}
+                topic={topic}
+                key={index}
+              />
+            ))}
           </div>
-          <Button onClick={handleAddPAT} type="primary" className="w-full mt-3 md:mt-0 h-10">
-            Continue
-          </Button>
         </div>
+        <Button onClick={handleUpdateInterest} type="primary" className="w-full mt-3 md:mt-0 h-10">
+          Confirm Selections
+        </Button>
       </div>
     </>
   );
 };
 
 interface LoginStep3Props {
   handleLoginStep: handleLoginStep;
-  checkFollowed: {
-    isClickedFollowed: boolean;
-    setIsClickedFollowed: React.Dispatch<React.SetStateAction<boolean>>;
-  };
-  repoList: LoginRepoObjectInterface[];
+  interests: string[];
 }
 
-const LoginStep3: React.FC<LoginStep3Props> = ({ repoList, checkFollowed }) => {
+const LoginStep3: React.FC<LoginStep3Props> = ({ interests }) => {
   captureAnayltics("User Onboarding", "onboardingStep3", "visited");
   const store = useStore();
   const router = useRouter();
   const { sessionToken } = useSupabaseAuth();
+  const [timezone, setTimezone] = useState("");
 
-  const [isFollowing, setIsFollowing] = useState<boolean[]>(repoList.map(() => false));
-  const following = isFollowing.filter((follow) => follow);
 
-  const handleSkipAddRepo = async () => {
+  const handleUpdateTimezone = async () => {
     try {
-      const selectedRepos = repoList.filter((_, index) => isFollowing[index]);
-      const repoIds = selectedRepos.map((repo) => repo.repoId as number);
-
-      await fetch(`${process.env.NEXT_PUBLIC_API_URL}/auth/onboarding`, {
+      const data = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/auth/onboarding`, {
         method: "POST",
         headers: {
           Authorization: `Bearer ${sessionToken}`
```

Comment:
This needs to include headers for JSON

```suggestion
          'Content-Type': 'application/json',
          Authorization: `Bearer ${sessionToken}`
```